### PR TITLE
Feature history endpoint

### PR DESCRIPTION
**Pull request by jatsqi**
- Add HistoryEndpoint
- Add tests
- Changes to the DAO ownership

### CODE REVIEW

File: src/main/kotlin/com/chillibits/coronaaid/repository/HistoryRepository.kt [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
 import com.chillibits.coronaaid.model.db.HistoryItem
+import com.chillibits.coronaaid.model.dto.HistoryItemDto
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 
-interface HistoryRepository: JpaRepository<HistoryItem, Int>
\ No newline at end of file
+interface HistoryRepository: JpaRepository<HistoryItem, Int> {
+
+    @Query("SELECT it FROM HistoryItem it, Infected i WHERE it.infectedId = i.id AND i.id = ?1")
```
Comment (marcauberer) [inline src/main/kotlin/com/chillibits/coronaaid/repository/HistoryRepository.kt:10]: Does this also work with "SELECT it FROM HistoryItem it WHERE it.infectedId = ?1"?

Comment (jatsqi) [reply src/main/kotlin/com/chillibits/coronaaid/repository/HistoryRepository.kt:10]: Most likely. I copied this code from the TestRepository, so we could change it there too.

Comment (marcauberer) [reply src/main/kotlin/com/chillibits/coronaaid/repository/HistoryRepository.kt:10]: Yes

Comment (marcauberer) [reply src/main/kotlin/com/chillibits/coronaaid/repository/HistoryRepository.kt:10]: Could you change that in both repositories before we merge this pr?

Comment (jatsqi) [reply src/main/kotlin/com/chillibits/coronaaid/repository/HistoryRepository.kt]: Done

**Comment by marcauberer** [comment]
My pr is ready. Let's merge it first.

**Comment by jatsqi** [comment]
I will fix this

**Pull request merged**